build.gradle

```diff
@@ -47,9 +49,12 @@ dependencies {
     implementation "io.prometheus:simpleclient_servlet:${prometheusClientVersion}"
     implementation "io.prometheus:simpleclient_hotspot:${prometheusClientVersion}"
 
-    //
+    //logging deps
     implementation "org.slf4j:slf4j-api:${project.ext.slf4jVersion}"
-    implementation "org.slf4j:slf4j-jdk14:2.0.0-alpha1"
+    implementation "org.apache.logging.log4j:log4j-slf4j18-impl:${log4jVersion}"
+    implementation "org.apache.logging.log4j:log4j-api:${log4jVersion}"
+    implementation "org.apache.logging.log4j:log4j-core:${log4jVersion}"
```

Comment:
You also need to add the dependency for lmax disruptor to make the logging async http://logging.apache.org/log4j/2.x/manual/async.html